app/Http/Livewire/ToursTable.php

```diff
@@ -12,10 +12,26 @@ class ToursTable extends Component
 
     public $search = '';
 
+    public $sortField;
+    public $sortDirection = 'asc';
+
+    public function sortBy($field) {
+        if ($this->sortField === $field) {
+            $this->sortDirection = $this->sortDirection === 'asc' ? 'desc' : 'asc';
+        } else $this->sortDirection = 'asc';
+        $this->sortField = $field;
+    }
+    
+
+    public function updatingSearch()
+    {
+        $this->resetPage();
+    }
+
     public function render()
     {
         return view('livewire.tours-table', [
-            'tours' => Tour::search('destination', $this->search)->paginate(5)
+            'tours' => Tour::search('destination', $this->search)->orderBy($this->sortField, $this->sortDirection)->paginate(10)
```

Comment:
This is giving me an error upon initial page load. Says `Column not found: 1054 Unknown column '' in 'order clause' `.
How did yours get past this error to display anything?